src/Peering/Peering/ChangeLog.md

```diff
@@ -18,6 +18,11 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Bug fix in set connection for exchange that caused CDIR notation
+* Allowed adding connections during legacy convert
+* Added tests for set Exchange operations
+
+## version 0.2.0
```

Comment:
@rileymckenna you can remove this line (and the empty line above it) and we'll bundle the changes in this PR with the ones going out in the release